apps/desktop/src/renderer/index.css

```
@@ -425,15 +427,37 @@ h6 {
+
+.ade-shell-header-utility-zoom {
+  height: var(--shell-header-utility-h);
+  min-width: 28px;
+  padding-inline: 4px;
+  font-size: 9px;
+  line-height: 1;
+}
+
 .ade-shell-project-tab {
   --project-tab-accent: var(--color-accent);
+  height: var(--shell-project-tab-height);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)